src/router-view.ts

```diff
@@ -0,0 +1,236 @@
+import { Container } from 'aurelia-dependency-injection';
+import { createOverrideContext, OverrideContext } from 'aurelia-binding';
+// tslint:disable-next-line:max-line-length
+import { ViewSlot, ViewLocator, customElement, noView, BehaviorInstruction, bindable, CompositionTransaction, CompositionEngine, ShadowDOM, SwapStrategies, ResourceDescription, HtmlBehaviorResource, CompositionTransactionNotifier, View, CompositionTransactionOwnershipToken, Controller, ViewFactory, CompositionContext } from 'aurelia-templating';
+import {
+  Router,
+  ViewPortInstruction,
+  ViewPort
+} from 'aurelia-router';
+import { Origin } from 'aurelia-metadata';
+import { DOM } from 'aurelia-pal';
+
+class EmptyViewModel {
+
+}
+
+export class RouterView implements ViewPort {
+
+  static inject() {
+    return [DOM.Element, Container, ViewSlot, Router, ViewLocator, CompositionTransaction, CompositionEngine];
+  }
+
+  swapOrder: string;
```

Comment:
is this part of an enum somewhere

Reply:
I wasn't sure about backward compat for this so string seemed to be safe enough

Reply:
On a second thought, this package are probably never touch directly in application, it could be fine to change to enum